launchable/test_runners/cypress.py

```diff
@@ -0,0 +1,39 @@
+import click
+import os
+import sys
+from . import launchable
+from junitparser import TestSuite, JUnitXml
+
+
+@click.argument('reports', required=True, nargs=-1)
+@launchable.record.tests
+def record_tests(client, reports):
+    for r in reports:
+        client.report(r)
+
+    def testsuites(xml) -> [TestSuite]:
+        testsuites = []
+        if isinstance(xml, JUnitXml):
+            filepath = xml._elem.find(
+                './/testsuite[@name="Root Suite"]').get("file")
+            for suite in xml:
+                suite._elem.attrib.update(
```

Comment:
here is the main part of a cypress plugin. 
Get a `file` attribute value used by XPath and add it to a testsuite attribute for each testsuites, likes this (based on https://github.com/launchableinc/cli/pull/101/files#diff-57f8b2c67ad176f3307742a51acc476264050670ebe13c6492441fd17596f971).

```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="1.1780" tests="6" failures="0">
  <testsuite name="Root Suite" timestamp="2021-01-20T08:49:02" tests="0" file="cypress/integration/examples/window.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Window" timestamp="2021-01-20T08:49:02" tests="3" time="0.3150" failures="0">
    ...
  </testsuite>
</testsuites> 
```
↓
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="1.1780" tests="6" failures="0">
  <testsuite name="Root Suite" timestamp="2021-01-20T08:49:02" tests="0" file="cypress/integration/examples/window.spec.js" time="0.0000" failures="0" filepath="cypress/integration/examples/window.spec.js">
  </testsuite>
  <testsuite name="Window" timestamp="2021-01-20T08:49:02" tests="3" time="0.3150" failures="0" filepath="cypress/integration/examples/window.spec.js">
    ...
  </testsuite>
</testsuites> 
```

But, I'm may sure there's a better way...